packages/ember-htmlbars/tests/helpers/new_each_helper_test.js

```diff
@@ -0,0 +1,334 @@
+/*jshint newcap:false*/
+import Ember from "ember-metal/core"; // Ember.lookup;
+import EmberObject from "ember-runtime/system/object";
+import run from "ember-metal/run_loop";
+import EmberView from "ember-views/views/view";
+import _MetamorphView from "ember-views/views/metamorph_view";
+import { computed } from "ember-metal/computed";
+import ArrayController from "ember-runtime/controllers/array_controller";
+import { A } from "ember-runtime/system/native_array";
+import { default as EmberController } from "ember-runtime/controllers/controller";
+import { Registry } from "ember-runtime/system/container";
+
+import { get } from "ember-metal/property_get";
+import { set } from "ember-metal/property_set";
+import { runAppend, runDestroy } from "ember-runtime/tests/utils";
+
+import compile from "ember-template-compiler/system/compile";
+
+var people, view, registry, container;
+
+// This function lets us write {{#EACH|people|p}} {{p}} {{/each}}
+// and generate:
+//
+// - {{#each p in people}} (legacy)
+// - {{#each people as |p|}} (legacy)
```

Comment:
why is `{{#each people as |p|}}` marked as legacy? Are we not keeping that syntax?